src/pages/Review/Star.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './Review.scss';
+
+class Star extends Component {
+  render() {
+    const { active } = this.props;
+    return (
+      <>
+        <span className={active ? 'active' : ''} key={this.props.el}>
```

Comment:
```suggestion
        <span className={active ? 'active' : ''}>
```
- 구조분해 할당 해주시는데 여기는 Key prop 들어갈 자리가 아닌거 같아서 그냥 삭제해주시면 될거 같습니다

Reply:
수정완료했습니다!